runner/runner.go

```diff
@@ -43,7 +43,7 @@ import (
 	batchrouterutils "github.com/rudderlabs/rudder-server/router/utils"
 	"github.com/rudderlabs/rudder-server/services/alert"
 	"github.com/rudderlabs/rudder-server/services/archiver"
-	"github.com/rudderlabs/rudder-server/services/controlplane/features"
+	"github.com/rudderlabs/rudder-server/services/controlplane/client"
```

Comment:
I want to avoid `cp` repetition in the import path. 

Having a package named just `client` is not uncommon in go. I believe it is ok since it is under the context of `controlplane`. A similar pattern is used by the [etcd client](https://pkg.go.dev/go.etcd.io/etcd/client/v3) as well.